devices/schneider_electric.js

```diff
@@ -296,11 +296,17 @@ module.exports = [
         description: 'LK FUGA wiser wireless battery 4 button switch',
         fromZigbee: [fz.command_on, fz.command_off, fz.command_move, fz.command_stop],
         toZigbee: [],
-        exposes: [e.action(['on', 'off', 'brightness_*'])],
+        endpoint: (device) => {
+            return {'s1': 21, 's2': 22};
```

Comment:
`s1` and `s2` are non valid endpoint names, can you change this to `l1` and `l2`? These are allowed: https://github.com/Koenkk/zigbee2mqtt/blob/9bd46932c365e886ed0b1f9b8740d8cce632230c/lib/util/utils.js#L34

Reply:
I can do that, no problem.  Alternatively, would `top` and `bottom` be preferable?  This is how the button pairs physically relate to the remote unit.

Reply:
Yes, `top` and `bottom` is better.

Reply:
Great, I've now used those.